### ISSUE TITLE
Multiline tablecells with hardbreak exported incorrectly

### ISSUE THREAD

**Issue by neopostmodern** [aguingand/tiptap-markdown#54]
**Describe the bug**
The markdown generated for a table with a hardbreak inside a cell is incorrect and can't properly be re-imported.

**To Reproduce**
1. Create a table with a hardbreak inside a table cell or use this markdown table:
  ```
| Table <br/> cell | Test |
| --- | --- |
| 1 | 2 |
```
  ![image](https://github.com/aguingand/tiptap-markdown/assets/786840/a80d271f-d7a3-40a2-9601-45c16c3b0783)

2. Export the table to Markdown, you get:
  ```
| Table \
cell | Test |
| --- | --- |
| 1 | 2 |
```
3. Re-import:
  ![image](https://github.com/aguingand/tiptap-markdown/assets/786840/f887fb11-d7cf-48f7-bf3a-165d9586d98c)

**Expected behavior**
The export should be correct markdown and re-importable – I personally think `<br />` would be the best in this case.

**Screenshots**
See above.

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04
 - Browser: Electron
 - Version: 24, like Chrome 112

**Edited to add:** this happens independently of the `html` setting

**Comment by aguingand** [comment]
Released in v0.8.9

**Comment by neopostmodern** [comment]
Amazing, thank you @aguingand !
